doc/llvm_obfuscation_tutorial.rst

```diff
@@ -505,6 +523,8 @@ To prevent overflowing we have set the maximum for Any1 and Any2 to 10, but this
 We need to make sure that x and y are not too big. The trick is that we have no information on their value at compile time.
 The solution we chose is to apply a bitmask to x and y in order to obtain a variable of which we know the max value.
 
+The careful reader may have noticed that randomly picking from ``IntegerVect`` has we did not check for uniqueness of its elements ;-)
```

Comment:
On Mon, Feb 02, 2015 at 06:00:33AM -0800, Merlini Adrien wrote:
> In doc/llvm_obfuscation_tutorial.rst:
> > @@ -505,6 +523,8 @@ To prevent overflowing we have set the maximum for Any1
> and Any2 to 10, but this
> >  We need to make sure that x and y are not too big. The trick is that we have
> no information on their value at compile time.
> >  The solution we chose is to apply a bitmask to x and y in order to obtain a
> variable of which we know the max value.
> >
> > +The careful reader may have noticed that randomly picking from
> ``IntegerVect`` has we did not check for uniqueness of its elements ;-)
> typo I guess 'has' -> 'as'.
> But I don't understand your sentence, is there a part missing?

Thanks for the typo. The goal is to state thatwe're using a vector to
store a set of values, then pick a random one but the very same value
could appear several times in the vector. there was indeed a missing
word. This should be fixed.